index.html

```diff
@@ -74,5 +76,179 @@ <h3>   I am a Software Developer </h3>
 
       </div>
     </div>
+<section class="work-section">
+  <header class="work-section-heading">
+    <h2>My Recent Works</h2>
+    <div class="line"></div>
+    <!-- <hr class="hr"> -->
+  </header>
+  <div class="work-section-image">
+    <img class="img" src="./Images/image1.jpg" alt="">
+  </div>
+  <section class="work-section-texts">
+      <h3 class="work-section-text1">Multi-Post Stories</h3>
+      <p class="paragraph">
+        A daily selection of privately personalized reads;
+         no accounts or sign-ups required. has been the industry's
+          standard dummy text ever since the 1500s,
+         when an unknown printer took a standard dummy text.
+      </p>
+      <div class="work-section-button">
+        <button type="button" class="btn1"><span>Css</span></button>
+        <button type="button" class="btn2"><span>Html</span></button>
+        <button type="button" class="btn3"><span class="big-span">bootstrap</span></button>
+        <button type="button" class="btn4"><span>Ruby</span></button>
```

Comment:
- [ ] [Optional] I will suggest you use the same class for the CSS styling, the use of classes in styling is for reusing the same style between different components that looks the same.